src/core/stream.h

```diff
@@ -448,6 +460,12 @@ QuicStreamSendGetState(
 {
     if (Stream->Flags.LocalNotAllowed) {
         return QUIC_STREAM_SEND_DISABLED;
+    } else if (Stream->Flags.LocalCloseResetReliable) {
+        if (Stream->Flags.LocalCloseResetReliableAcked) {
+            return QUIC_STREAM_SEND_RELIABLE_RESET_ACKED;
+        } else {
+            return QUIC_STREAM_SEND_RELIABLE_RESET;
```

Comment:
Why is this line never hit in tests? Is this state actually valid?

Reply:
This state is when we have set reliableOffset and called shutdown, but awaiting the receiver to acknowledge that we are reliably shutting down. Kind of like an intermediary state.